### PR TITLE
Add support for GHE.

### PR DESCRIPTION
**Pull request by annitya**
Code isn't perfect, but definitely gets the job done.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,10 +14,13 @@ You need a `~/.jump.json` config file that looks like this:
 {
   "config": {
     "github_access_token": "youraccesstokenhere",
-    "globalShortcut": "Command+J"
+    "globalShortcut": "Command+J",
+    "host": "optional",
+    "pathPrefix": "optional"
```
Comment (davegaeddert) [inline README.md]: Could we put these under "githubEnterprise" to make it a little more explicit what it is for?

```json
"githubEnterprise": {
  "host": "optional",
  "pathPrefix": "optional"
}
```

File: src/github.js [hunk 2]
```diff
@@ -2,22 +2,35 @@ import GitHubApi from 'github'
 import { getUserSettings } from './utils'
 
 export const loadUserRepos = (callback) => {
-    const userSettings = getUserSettings()
-    if (!userSettings) return
+    const userSettings = getUserSettings();
```
Comment (davegaeddert) [inline src/github.js]: Could you remove the extra `;` that got added, just to keep consistency with the rest of the code (which I don't think has them right now)?

**Comment by annitya** [comment]
I wanted to use the spread-operator instead of Object.assign. Makes for slightly prettier code, but the babel-transform was missing. Decided against adding it to keep the pull-request simple.

**Comment by davegaeddert** [comment]
Ok, yeah I think this is easier for now. I do have some other things I want to do with this at some point so we'll probably add babel-transform later.

Thanks for the help!

**Pull request merged**